Gemfile.lock

```diff
@@ -6,14 +6,14 @@ GEM
     asciidoctor (2.0.15)
     colorator (1.1.0)
     concurrent-ruby (1.1.9)
-    em-websocket (0.5.2)
+    em-websocket (0.5.3)
       eventmachine (>= 0.12.9)
-      http_parser.rb (~> 0.6.0)
+      http_parser.rb (~> 0)
```

Comment:
Huh? Is this right @mudge ?

Reply:
[Looks like it](https://rubygems.org/gems/em-websocket/versions/0.5.3): `em-websocket` has a dependency on _any_ version of `http_parser.rb`.